packages/notification-center/src/i18n/languages/pu.ts

```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const PU: ITranslationEntry = {
+  translations: {
+    notifications: 'ਸੂਚਨਾਵਾਂ',
+    markAllAsRead: ' ਸਭ ਨੂੰ ਪੜ੍ਹਿਆ ਹੋਇਆ ਮਾਰਕ ਕਰੋ',
+    poweredBy: 'ਦੁਆਰਾ ਸੰਚਾਲਿਤ',
+    settings: 'ਸੈਟਿੰਗਾਂ',
+  },
+  lang: 'pu',
+};
```

Comment:
Please remove this file as it is related to Punjabi language code.

Reply:
@p-fernandez  deleted. Kindly merge